deploy/nais.yml

```diff
@@ -15,8 +15,13 @@ spec:
     limits:
       memory: 2048Mi
   replicas:
-    max: 2
     min: 2
+    max: 3 # Ønsker vi å øke denne, må vi først øke antall Kafka-partisjoner
+    scalingStrategy:
+      kafka:
+        consumerGroup: helsearbeidsgiver-bro-sykepenger-v1
+        topic: helsearbeidsgiver.pri
+        threshold: 10
```

Comment:
Hvordan fungerer dette egentlig? Er det slik at det scaleres opp og ned når offsetlag går over / under 10? Eller må det være over 10 over en viss tid ⏲️ ?

Reply:
Detaljene er gjemt i Nais, men i bunn og grunn fungerer det som du antar. Ut i fra k8s-doken så ser det ut som man kan spisse konfigurasjonen for å ha mer kontroll på selve opp- og nedskaleringen, men antar jeg at Nais-gjengen har satt fornuftige verdier for.